lib/transport/selenium-webdriver/method-mappings.js

```diff
@@ -1,6 +1,8 @@
 const {WebElement, Origin, By} = require('selenium-webdriver');
 const {Locator} = require('../../element');
 const {isString} = require('../../utils');
+const fs = require('fs');
+const cdp = require('./cdp');
```

Comment:
```suggestion
const cdp = require('./cdp.js');
```

Reply:
Done. But was just wondering, why add `.js` here?

Reply:
Just to make it clear that it is a file we're importing.